docs/reference/search/count.asciidoc

```diff
@@ -1,11 +1,91 @@
 [[search-count]]
 === Count API
 
-The count API allows to easily execute a query and get the number of
-matches for that query. It can be executed across one or more indices.
-The query can either be provided using a simple query string as a
-parameter, or using the <<query-dsl,Query DSL>> defined within the request
-body. Here is an example:
+Gets the number of matches for a search query.
+
+[source,js]
+--------------------------------------------------
+GET /twitter/_count?q=user:kimchy
+--------------------------------------------------
+//CONSOLE
+
+NOTE: The query being sent in the body must be nested in a `query` key, same as
+the <<search-search,search api>> works.
+
+
+[[search-count-api-request]]
+==== {api-request-title}
+
+`PUT /<index>/_count`
+
+
+[[search-count-api-desc]]
+==== {api-description-title}
+
+The count API allows you to execute a query and get the number of matches for 
+that query. It can be executed across one or more indices. The query can either 
+be provided using a simple query string as a parameter, or using the 
+<<query-dsl,Query DSL>> defined within the request body.
+
+The count API can be applied to <<search-multi-index,multiple indices>>.
+
+The operation is broadcast across all shards. For each shard id group, a replica 
+is chosen and executed against it. This means that replicas increase the 
+scalability of count.
+
+
+[[search-count-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
+
+
+[[search-count-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=analyzer]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=analyze_wildcard]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=default_operator]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=df]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
+
+`ignore_throttled`::
```

Comment:
Can we move this to `common-parms.asciidoc`?

Looks like this parameter is also used here:

- https://www.elastic.co/guide/en/elasticsearch/reference/master/search-template.html#search-template-api-query-params
- https://www.elastic.co/guide/en/elasticsearch/reference/master/search-search.html#search-search-api-query-params